### PR TITLE
Update commands' help to match the text in ecs cli documentation.

### PR DESCRIPTION
**Pull request by uttarasridhar**
Reference URL : http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_CLI_reference.html

### CODE REVIEW

File: ecs-cli/modules/command/cluster.go [hunk 1]
```diff
@@ -45,47 +45,47 @@ func UpCommand() cli.Command {
...
 			},
 			cli.StringFlag{
 				Name:  securityGroupFlag,
-				Usage: "[Optional] Specify an existing security group to associate it with container instances. Defaults to creating a new one.",
+				Usage: "[Optional] Specifies an existing security group to associate with your container instances. If you do not specify a security group here, then a new one is created.",
 			},
 			cli.StringFlag{
 				Name:  sourceCidrFlag,
-				Usage: "[Optional] Specify a CIDR/IP range for the security group to use for container instances in your cluster. Defaults to 0.0.0.0/0 if --security-group is not specified",
+				Usage: "[Optional] Specifies a CIDR/IP range for the security group to use for container instances in your cluster. This parameter is ignored if an existing security group is specified with the --security-group option. Defaults to 0.0.0.0/0 .",
```
Comment (euank) [inline ecs-cli/modules/command/cluster.go]: There's an extra space by the punctuation.

File: ecs-cli/modules/compose/cli/ecs/app/command.go [hunk 2]
```diff
@@ -51,7 +51,7 @@ const (
 func ComposeCommand(factory ProjectFactory) cli.Command {
 	return cli.Command{
 		Name:   "compose",
-		Usage:  "Execute docker-compose style commands on an ECS cluster.",
+		Usage:  "Executes docker-compose–style commands on an ECS cluster.",
```
Comment (euank) [inline ecs-cli/modules/compose/cli/ecs/app/command.go]: The added hyphen there is an en-dash, not hyphen, which can look odd in some terminals.

**Comment by euank** [comment]
:+1:

**Comment by uttarasridhar** [comment]
merged as 6e651e9c52c41420734ca8692ed154edeeabae57

**Pull request closed without merging**